stacker/providers/aws/interactive.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import time
+
+import yaml
+
+from ... import exceptions
+from .default import (
+    Provider as BaseProvider,
+    retry_on_throttling,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def get_change_set_name():
+    """Return a valid Change Set Name.
+
+    The name has to satisfy the following regex:
+        [a-zA-Z][-a-zA-Z0-9]*
+
+    And must be unique across all change sets.
+
+    """
+    return 'change-set-{}'.format(int(time.time()))
+
+
+class Provider(BaseProvider):
```

Comment:
just falls back to the default AWS provider right now.

I figured the change sets were only really useful when updating